stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -66,6 +61,11 @@ spec:
       - name: {{ template "redis.fullname" . }}
         image: "{{ template "redis.image" . }}"
         imagePullPolicy: {{ default "" .Values.image.pullPolicy | quote }}
+        {{- if .Values.master.securityContext.enabled }}
+        securityContext:
+          fsGroup: {{ .Values.master.securityContext.fsGroup }}
```

Comment:
I just recall that `fsGroup` should be specified at pod level and it will fail if you specify it for a container. 

I guess you could split the securityContext configuration and specify `fsGroup` at pod level and `runAsUser` at container level but I don't know if that would work with your Istio setup.